File: src/main/java/main/utils/json/restrictedchannels/RestrictedChannelsConfigField.java
package main.utils.json.restrictedchannels;

import main.utils.json.GenericJSONField;

public enum RestrictedChannelsConfigField implements GenericJSONField {
    VOICE_CHANNELS("voice_channels"),
    TEXT_CHANNELS("text_channels");

    private final String str;

    RestrictedChannelsConfigField(String str) {
        this.str = str;
    }

    @Override
    public String toString() {
        return str;
    }
}

File: src/main/kotlin/main/utils/json/restrictedchannels/RestrictedChannelsConfigField.kt
package main.utils.json.restrictedchannels

import main.utils.json.GenericJSONField

enum class RestrictedChannelsConfigField(private val str: String) : GenericJSONField {
    VOICE_CHANNELS("voice_channels"),
    TEXT_CHANNELS("text_channels");

    override fun toString(): String = str
}